code-challenges.js

```diff
@@ -31,24 +41,97 @@ const temperature2 = 350
 const temperature3 = 212
 // Expected output: "212 is at boiling point"
 
+
+const boilingPoint = (num1) => {
+    if (num1 < 212) {
+    return `${num1} is below boiling point`
+    }
+    else if (num1 > 212) {
+        return `${num1} is above boiling point`
+    }
+    else {
+        return `${num1} is at boiling point`
+    }
+}
+
+console.log(boilingPoint(temperature1))
+//output: 42 is below boiling point
+
+console.log(boilingPoint(temperature2))
+//output: 350 is above boiling point
+
+console.log(boilingPoint(temperature3))
+//output: 212 is at boiling point
+
+
+
+
+
 // --------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
 // Pseudo code:
 
+// Name: combinedArrayLength
+// input: const padres1984WorldSeriesRuns, const padres1998WorldSeriesRuns 
+// output: length of array1 and array2 combined 
+// process: create a new variable named combinedArrayLength that will combine the length of array1 and array2 and return a number datatype. (.length method)
+
 const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
 const padres1998WorldSeriesRuns = [6, 3, 5, 3]
 // Expected output: 9
 
+const combinedArrayLength = padres1984WorldSeriesRuns.length + padres1998WorldSeriesRuns.length
+
+console.log(combinedArrayLength)
+//output: 9
+
+
+
+
+
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
 
 // Pseudo code:
 
+// Name: reverseString
+// input: currentCohort
+// process: create a new variable named reverseString and set it equal to the const variable named currentCohort and use the following built in methods to work through - .split("") to separate each character then .reverse() to reverse the characters and finall .join("") to join each character back together.
+// .split("")
+    // output "C h a r l i e  2 0 2 3"
+// . reverse()
+    // output "3 2 0 2  e i l r a h C"
+// .join("")
+    //output "3202 eilrahC"        
+
 const currentCohort = "Charlie 2023"
 // Expected output: "3202 eilrahC"
 
+//const reverseString = currentCohort.split("").reverse().join("")
+
+//console.log(reverseString)
+// output: "3202 eilrahC"
+
+//OR
+
+const reverseString = (backwards) => {
+    return backwards.split("").reverse().join('')
+}
+
```

Comment:
Love that you went back for a refactor! This is great! Next time I would also add some pseudo code to show that you know what is going on for the refactor as well!